### DISCUSSION TITLE
Inserting callout or block quote into outline does not work.

### DISCUSSION THREAD

**Discussion by chrjohns** [RyotaUshio/obsidian-pdf-plus#196] [Q&A]
I have not been able to figure out how to insert a pdf++ quote or callout into the nested hierarchy of an outline so that it displays correctly.  Normal callouts work, and block quotes work. Is this functionality available?  Any suggestions, or is this just not possible at this time?

**Comment by RyotaUshio** [comment]
Unfortunately, it's not possible currently because PDF++ does not know where the text will be inserted to at the moment the copy occurs. But any PR will be welcomed:)
